requirements/base-requirements.txt

```diff
@@ -266,7 +266,7 @@ oauthlib==3.1.0
     # via
     #   django-oauth-toolkit
     #   requests-oauthlib
-openpyxl==2.6.4
+openpyxl==3.0.6
```

Comment:
There may be a bug in our requirements building automation (`make requirements`), or a mistake was made somewhere? #29081 was merged on Feb 7th after #29141 was created, but before it was merged. When @czue tried to reconcile the conflict he updated [base-requirements.in](https://github.com/dimagi/commcare-hq/pull/29141/commits/f67ebf7ea6257db679ecc6a829f2c7405c671613#diff-b87f325c37ee3f502ee85c8014b25f13cf82f3ab93d971f8a715705ae3961f24), but it didn't get propagated to all requirements files. The commit message on https://github.com/dimagi/commcare-hq/pull/29141/commits/60de76f072886859027f9863a4e96e3adfec2b52 implies that he ran `make requirements`, but the only file updated was **docs-requirements.txt**, so something unexpected happened there.

Might be worth re-reviewing https://github.com/dimagi/commcare-hq/pull/29141/commits/f67ebf7ea6257db679ecc6a829f2c7405c671613 or maybe run diffs on requirements files before and after #29141 was merged to make sure everything got updated correctly.

Reply:
I think everything is fine from a practical perspective. 

The issue is that `make requirements` doesn't actually modify `base-requirements.txt` (https://github.com/dimagi/commcare-hq/blob/master/requirements/Makefile#L5-L9). I didn't bother adding it because the only reason it exists is due to this issue: https://github.com/dimagi/commcare-hq/pull/29141#issuecomment-776690666. So all requirements files that are *used* are correct.

That said, probably no downside in adding a `base-requirements` build to `make requirements` so that we don't get confused by this behavior in the future. I'll submit a PR for that now (won't bother touching dependencies so it doesn't conflict with this PR).

Reply:
https://github.com/dimagi/commcare-hq/pull/29202